EasyPost/EasyPost.csproj

```diff
@@ -47,14 +47,15 @@
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
     <SignAssembly>true</SignAssembly>
-    <AssemblyOriginatorKeyFile>..\EasyPost.pfx</AssemblyOriginatorKeyFile>
+    <DelaySign>true</DelaySign>
```

Comment:
Will discuss offline.

Reply:
As discussed, delay signing with only a public key allows better security practices. The public key can be shared and used by anyone to sign the DLL with the public key during the build, allowing only a select few people to have access to the private key to complete the signing process post-build, rather than requiring anyone building the project to have access to the private key during build.